src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,850 @@
+{
```

Comment:
<blockquote>in FAST-OAD(-GA)</blockquote><p>I would not want to bother you, but for this occurrence (and many others in the notebooks), as long it is about core features of FAST-OAD, I think we should stick to "FAST-OAD" only. (or else, we should also mention FAST-OAD-CS25, shouldn't we ?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Totally agreed, almost all of the mention to FAST-OAD(-GA) were actually a poor attempt at quoting both FAST-OAD and FAST-OAD(-GA). But since, the core has been isolated, and it thus makes a lot more sense to refer to the framework except for a couple of times. 
We now refer to FAST-OAD almost always but a particular attention will be paid to that during the final proofread
Corrected in 11d3a091 and 326f20b5